src/song/models.py

```diff
@@ -0,0 +1,11 @@
+from django.db import models
+
+# Create your models here.
+class Lyric(models.Model):
+    date_added = models.DateTimeField(auto_now_add=True)
```

Comment:
The `Lyric` model is supposed to relate to a song. I understand that as of now, we have no `Song` model. Do note however that in the future, we'll need to relate the `Lyric` model to the `Song` model if we implement a `Song` model.